theme/js/plugins/api.js

```diff
@@ -0,0 +1,15 @@
+import axios from "axios";
+import config from "../config";
+
+//Instanciate axios with base URL from config
+const api = axios.create({
+  baseURL: config.api_root,
+});
+
+//TODO, probably : add interceptors to add CSRF info and credentials
```

Comment:
I don't think so, credentials are handled through sessions and no CSRF on API endpoint (not sure).